client/onboarding-prototype/index.tsx

```diff
@@ -37,12 +42,36 @@ const Step: React.FC< Props > = ( { name, children } ) => {
 const OnboardingStepper = () => {
 	const { data } = useOnboardingContext();
 
-	const handleComplete = () => {
-		const { connectUrl } = wcpaySettings;
-		const url = addQueryArgs( connectUrl, {
-			progressive: fromDotNotation( data ),
+	const isEligibleForPo = async () => {
+		// TODO GH-5475 maybe move somewhere, not sure if it's the best place for this
```

Comment:
I think it should be moved somewhere but not sure where is better place 🙈

Reply:
As there is a loading screen ("Let’s get you setup for payments") in the design. I think we can create another `Step` and do the eligibility call and redirect there. You can do the logic and leave the design for #4746.